src/sql/src/plan/statement/ddl.rs

```diff
@@ -2914,9 +2960,10 @@ pub fn plan_drop_item(
     }
     if object_type != catalog_entry.item_type() {
         bail!(
-            "{} is not of type {}",
+            "{} is type {:?}, should be {:?}",
```

Comment:
If this is user-visible output it should not be debug formatted

Reply:
This should've already been reverted to the original error, I just needed this while I was figuring out that I had to implement PartialEq for my catalog type.